src/contacts.cs

```diff
@@ -289,8 +289,8 @@ interface CNContactKey {
 		NSString InstantMessageAddresses { get; }
 	}
 
-	[iOS (9,0), Mac (10,11)]
-	[BaseType (typeof (NSObject))]
+ 	[iOS (9,0), Mac (10,15), Watch (2,0)]
```

Comment:
The original `[iOS (9,0), Mac (10,11)]` does not have to change here, because
- the type was available in macOS 10.11
- `[Watch (2,0)]` is implied since the framework/bindings are present (and it's the minimum we support)